app/routers/attendance.py

```diff
@@ -23,10 +23,26 @@
 
 # 1. mock data를 사용하는 경우
 
-@router.post("/work/start", response_model=APIResponseModel, response_class=JSONResponse)
-async def work_start():
+@router.post("/check/in", response_model=AttendanceCheckInAPIResponseModel, response_class=JSONResponse)
```

Comment:
API 권장사항으로 URL에 명사를 권장하는 것으로 알고 있는데([예시](https://dev-cool.tistory.com/32)

동사로 작성하신 이유가 있을까요?

Reply:
컨트롤 자원의 경우, 동사로 사용할 수 있는데 출근 요청을 하는 것이 "출근 기록"을 컨트롤 자원으로 여기지 않는 것이 적합해 보입니다!

명사로 사용한다면 다음과 같이 변경하는 것이 적합할 것 같습니다! 

```suggestion
@router.post("/check-in", response_model=AttendanceCheckInAPIResponseModel, response_class=JSONResponse)
```